Dockerfile.slim

```diff
@@ -6,7 +6,7 @@ ENV KUBECTL_RELEASE=1.20.6
 ENV AWS_CLI_VERSION=1.18.152
 ENV AWS_CLI_S3_CMD=2.0.2
 ENV AWS_AIM_AUTHENTICATOR_VERSION=0.5.5
-ENV GOOGLE_CLOUD_SDK_VERSION=374.0.0
+ENV GOOGLE_CLOUD_SDK_VERSION=396.0.0
```

Comment:
Adjust [Dockerfile.ubuntu](https://github.com/spinnaker/clouddriver/blob/341706911c97af883d57e949fd20598472141dab/Dockerfile.ubuntu) as well please.

Reply:
Done. Also bumped the version again to 398.0.0.

PTAL.